spec/controllers/curation_concerns/generic_works_controller_spec.rb

```diff
@@ -56,10 +56,11 @@
         let(:presenter) { double }
         before do
           allow(controller).to receive(:presenter).and_return(presenter)
+          allow(controller).to receive(:authorize_workflow_role!).and_return(nil)
```

Comment:
Why are we doing this here?  Should this be reflected in a change to the context?